docs/gameplay-component.md

```diff
@@ -0,0 +1,41 @@
+# Children's BBC - Genie Gameplay Component
+
+* [How is Genie structured?](#how-is-genie-structured)
+* [How do I use the three core engines?](#how-do-i-use-the-three-core-engines)
+* [What coding and test standards do I need to apply?](#what-coding-and-test-standards-do-i-need-to-apply)
+* [How does my component plug into Genie?](#how-does-my-component-plug-into-genie)
+* [How do I test my component in the Genie framework?](#how-do-i-test-my-component-in-the-genie-framework)
+* [Are there any areas of existing Children’s game delivery standards I need to apply?](#are-there-any-areas-of-existing-children’s-game-delivery-standards-i-need-to-apply)
+* [How do I get my build onto Children’s platforms?](#how-do-i-get-my-build-onto-children’s-platforms)
+* [What acceptance tests will the BBC carry out?](#what-acceptance-tests-will-the-bbc-carry-out)
+* [What documentation do I need to supply?](#what-documentation-do-i-need-to-supply)
+* [How can I access support?](#how-can-i-access-support)
```

Comment:
Should this maybe live at the top level, as this question is not specific to the gameplay component?

Reply:
I agree. At the time of writing I wasn't sure if there were separate and specific support processes for creating a gameplay component and making a theme. Regardless, this should probably be at the top level of the repo.